files/fr/web/javascript/reference/operators/destructuring_assignment/index.html

```diff
@@ -2,10 +2,14 @@
 title: Affecter par décomposition
 slug: Web/JavaScript/Reference/Operators/Destructuring_assignment
 tags:
-  - ECMAScript 2015
-  - JavaScript
-  - Opérateur
-  - Reference
```

Comment:
I think Reference is missing in English and should not be deleted in fr (as far as I can tell when I'm looking at several pages in en-US / mdn/content)

Reply:
After a few thoughts (cf. other comment in the PR), I'd leave those in English (but you may still add "Reference" ;) )

Reply:
Updated with the missing "Reference" tag